client/src/assets/css/custom.scss

```diff
@@ -0,0 +1,29 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300;400;600;700&display=swap");
+
+body {
+  margin: 0;
+  font-family: "Nunito Sans", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+ monospace;
+}
+
+.custom-shade{
+  -webkit-box-shadow: 0 1px 3px 1px #949191;
+  box-shadow: 0 1px 3px 1px #949191;
+}
+
+.App{
+  margin:0;
+  padding: 0;
+}
+
+$yellow: #E2D13A;
+$red:#CE5353;
+$secondary:#D2D2D2;
+
+@import "../../../node_modules/bootstrap/scss/bootstrap";
```

Comment:
Why not at the beginning of the file?

Reply:
esta hecho de esa forma para poder modificar los colores que se necesita para el diseño. Por ejemplo si se necesita cambiar alguna propiedad de la librería se hace antes del import.

Reply:
Pero en ese caso que no el import de bootstrap quita los cambios de este archivo? por ejemplo la font family, la seteas aca pero bootstrap pone otra fuente, por fa confirma

Reply:
Si, es como decís. Esta pisando los estilos definidos. Tengo que bajar las los estilos que definí arriba. para que estén después del @import de bootstrap